src/Tickets/Commerce/Order_Modifiers/Traits/Coupons.php

```diff
@@ -273,7 +273,7 @@ protected function remove_coupon_use( int $coupon_id, int $quantity = 1 ) {
 			$existing_uses = $this->get_uses_meta( $coupon_id );
 
 			// Use max() to ensure we don't set the usage to a negative number.
-			$existing_uses->meta_value = max( (int) $existing_uses->meta_value - $quantity, 0 );
+			$existing_uses->meta_value = (int) max( (int) $existing_uses->meta_value - $quantity, 0 );
```

Comment:
This isn't technically needed. The `max()` function will return the original value type, so since all of the values provided to `max()` are already type `int`, `max()` can only ever return type `int`.